src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Assembler;
+
+
+use Surfnet\ServiceProviderDashboard\Application\Dto\ServiceStatusDto;
+use Surfnet\ServiceProviderDashboard\Application\Service\ServiceStatusService;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Service;
+use Symfony\Component\Routing\RouterInterface;
+
+class ServiceStatusAssembler
+{
+    /**
+     * The different status types used
+     */
+    const SERVICE_STATE_INTAKE_CONDUCTED = 'intake-conducted';
+    const SERVICE_STATE_ENTITY_ON_TEST = 'entity-on-test';
+    const SERVICE_STATE_REPRESENTATIVE_APPROVED = 'representative-approved';
+    const SERVICE_STATE_CONTRACT_SIGNED = 'contract-signed';
+    const SERVICE_STATE_PRIVACY_QUESTIONS = 'privacy-questions';
+    const SERVICE_STATE_PRODUCTION_CONNECTION = 'production-connection';
```

Comment:
In retrospect to my own naming in previous commits. This state could have been named:

`entity on production` to be more in line with the entity on test status. It's up to you what you want to do with this.

Reply:
I've left it this way